spec/index.bs

```diff
@@ -4390,6 +4486,34 @@ The {{GPUTextureUsage}} flags determine how a {{GPUTexture}} may be used after i
                 1. Set |t|.{{GPUTexture/dimension}} to |descriptor|.{{GPUTextureDescriptor/dimension}}.
                 1. Set |t|.{{GPUTexture/format}} to |descriptor|.{{GPUTextureDescriptor/format}}.
                 1. Set |t|.{{GPUTexture/usage}} to |descriptor|.{{GPUTextureDescriptor/usage}}.
+
+                1.
+                    <div class=compatmode>
+                        If |t|.{{GPUObjectBase/[[device]]}}.{{device/[[features]]}} does not [=list/contain=] {{GPUFeatureName/"core-features-and-limits"}}:
+
+                        1. If |descriptor|.{{GPUTextureDescriptor/textureBindingViewDimension}} is [=map/exist|provided=]:
+                            1. Set |t|.{{GPUTexture/textureBindingViewDimension}} to |descriptor|.{{GPUTextureDescriptor/textureBindingViewDimension}}.
```

Comment:
I just noticed the spec only sets `t.textureBindingViewDimension` if it's provided explicitly, and not if it's defaulted. Will need to fix that.